apache/_docroot.shtml

```diff
@@ -1,19 +1,29 @@
 <div class="section">
     <h2>Can Tugboat find your document root?</h2>
     <p>
-        Tugboat looks for a folder named
-        <tt>docroot</tt> in your repository to serve content from. This is a common pattern, but many projects have their own
-        preference. If your document root is in a different location you have a couple of different options.
+        Tugboat looks for a folder named <code>docroot</code> in your repository to serve content from. This is a 
+        common pattern, but many projects have their own preference. If your document root is in a different location 
+        you have a couple of different options.
     </p>
-    <h3>1. Move your content to /docroot in your repository.</h3>
-    <p>This is the most straightforward fix for Tugboat, but can be problematic if your repository has already been deployed
-        elsewhere. If you choose this option, be sure to follow up on other deployments to make sure things still work.
+    <h3>Option 1. Add a <code>docroot/</code> folder to your repository</h3>
+    <p>
+      This is the most straightforward fix for Tugboat, but can be problematic if your repository has already been 
+      deployed elsewhere. If you choose this option, create a <code>docroot</code> folder and add your repository's web 
```

Comment:
"add" could be interpreted as needing to have multiple copies. Let's use "move" instead